api/projects/admin.py

```diff
@@ -9,6 +11,10 @@
 from projects.tags.models import Tag
 from segments.models import Segment
 
+if typing.TYPE_CHECKING:
```

Comment:
Why are we only importing them if TYPE_CHECKING?

Reply:
Because we only need them for typing.

Reply:
Is that the convention? Because both of them are going to be available at runtime anyway, and that if is a bit ugly? What do you think?

Reply:
Yeah, fair enough - I'm not too sure what the convention is. Seems like it should only really be used to avoid circular dependencies. I'll remove.